src/libsync/pushnotifications.cpp

```diff
@@ -14,6 +15,13 @@ PushNotifications::PushNotifications(Account *account, QObject *parent)
     : QObject(parent)
     , _account(account)
 {
+    connect(&_pingTimer, &QTimer::timeout, this, &PushNotifications::pingWebSocketServer);
+    _pingTimer.setSingleShot(true);
+    _pingTimer.setInterval(PING_INTERVAL);
+
+    connect(&_pongTimedOutTimer, &QTimer::timeout, this, &PushNotifications::onPingTimedOut);
```

Comment:
I'd say it should be named _pingTimedOutTimer? Or the slot be onPongTimeout?

I'd lean toward the former since the interval used it also PING_INTERVAL... that said I guess it's just because they are somehow on the same duration but could be different. It's a case where the "don't use magic numbers" is a bit confusing because a name is recycled while conceptually it shouldn't.